app/assets/javascripts/incidents.subquestions.js

```diff
@@ -75,4 +78,19 @@ $(function(){
     $('#multiple-agencies-info-text').toggle($('#screener_multiple_agencies_true').is(':checked'));
     $('#shots-fired-info-text').toggle($('#screener_shots_fired_true').is(':checked'));
   }).change();
+
+
+  function medicalAid() {
+    $('.medical-aid-received input[type="radio"]').on('click', function () {
+      togglePreExistingCondition()
+    })
+  }
+
+  function togglePreExistingCondition() {
+    if ($('input[value="No medical assistance or refused assistance"]').is(':checked')) {
+      $('.additional-question.pre-existing-condition').hide()
```

Comment:
Instead of showing and hiding, we enable and disable. Specifically, you can use the `enableQuestions('.additional-question.pre-existing-condition')` function (from util.js) as well as `disableQuestions`.